README.md

```diff
@@ -11,6 +11,19 @@ Our hotel booking project with databases
 - [Badges](#badges)
 - [License](#license)
 
+## Introduction
+
+Nowadays, tourists are becoming more and more. And hotel reservations are always relevant and necessary for people. Therefore, we want to develop a convenient database with useful information and rich functionality.
```

Comment:
> Nowadays, tourists are becoming more and more

Not bad (online translators say, that it's a good sentence), but for Me it was not that easy to understand this line of words (even with realising of what You wanted to say). To be more specific, I will say, that this sentence needs to be reworked (maybe, like, instead of "becoming", You can write "growing")

Reply:
I understood. I will add "number of tourists"

Reply:
to better understand, what I meant, in My head translation was,like:"туристи стають все більше й більше"